code/datums/components/shuttle_cling.dm

```diff
@@ -76,7 +78,7 @@
 
 ///Check if we're "holding on" to the shuttle
 /datum/component/shuttle_cling/proc/is_holding_on(atom/movable/movee)
-	if(movee.pulledby || !isturf(movee.loc))
+	if(movee.pulledby)
```

Comment:
Probably fixed it here, because if you are in a jaunt holder your loc isn't a turf

Reply:
Thats the check I removed though